src/config/nfs.md

```diff
@@ -0,0 +1,142 @@
+# Network Filesystems
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` package.
+
+NFS shares are configured with
+[nfs.conf(5)](https://man.voidlinux.org/nfs.conf.5).
+
+Before mounting an NFS share, it is necessary to
+[enable](./services/index.md#enabling-services) the `statd` and `procbind`
+services. If the server supports `nfs4`, the `statd` service isn't necessary,
+and `nfs4` should be used in the place of `nfs` as the mount type. To mount the
+NFS share, use the command below, where `host` is the hostname or IP address of
+the server. Mounting options can be found in
+[mount.nfs(8)](https://man.voidlinux.org/mount.nfs.8), while unmounting ones can
+be found in [umount.nfs(8)](https://man.voidlinux.org/umount.nfs.8).
+
+```
+# mount -t nfs host:/path/to/sourcedir /path/to/destdir
+```
+
+For example, if you wanted to connect `/volume` on a server at IP Address
+`192.168.1.99` to a directory `/mnt/volume` on your local system, you could do:
+
+```
+# mkdir /mnt/volume
+# mount -t nfs 192.168.1.99:/volume /mnt/volume
+```
+
+To have the directory mounted again when the system reboots, add an entry to
+[fstab(5)](https://man.voidlinux.org/fstab.5):
+
+```
+192.168.1.99:/volume /mnt/volume nfs rw,hard,intr 0 0
+```
+
+Check out [nfs(5)](https://man.voidlinux.org/nfs.5) for more information about
+the available options.
+
+Due to [void-runit#16](https://github.com/void-linux/void-runit/issues/16), if
```

Comment:
i would strongly prefer that https://github.com/void-linux/void-runit/issues/16 be resolved, rather than documenting workarounds in the Handbook; imo, the latter should be a last resort.